docs/terry-feng-GSoC-2024.md

```diff
@@ -0,0 +1,125 @@
+# Terry Feng - Google Summer of Code 2024
+
+## Overview 
+**Contributor**: Terry Feng ([@terryzfeng](https://github.com/terryzfeng)) <br>
```

Comment:
`<br>` can be replaced with two spaces, I think.

Reply:
Yeah, I made it this way so it would be single spaced rather than double spaced independent paragraphs.

preview: https://github.com/terryzfeng/web-audio-samples/blob/main/docs/terry-feng-GSoC-2024.md

Reply:
Okay. LGTM.